tests/js/profile/wearable/widget/wearable/Scrollview/Scrollview.js

```diff
@@ -44,6 +44,85 @@
 			helpers.restoreStub(ns.support.shape, "circle");
 		});
 
+		QUnit.test("_init", 4, function (assert) {
+			var element = document.getElementById("scrollview-page-w"),
+				scrollview = new Scrollview();
+
+			helpers.stub(Scrollview.prototype, "_setBouncingEffect", function () {
```

Comment:
This stub works but we rather make stab from instance method instead prototype method.

Reply:
Thank you for review.
Done.